WordsTableTableViewController.swift

```diff
@@ -0,0 +1,55 @@
+//
+//  WordsTableTableViewController.swift
+//  Swift Vocabulary
+//
+//  Created by Clean Mac on 4/8/20.
+//  Copyright © 2020 LambdaStudent. All rights reserved.
+//
+
+import UIKit
+
+
+
+class WordsTableTableViewController: UITableViewController {
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        var vocabWords: [VocabularyWord] = [
+        VocabularyWord(word: "word1", definition: "Def1"),
+        VocabularyWord(word: "word2", definition: "def2"),
+        VocabularyWord(word: "word3", definition: "def3"),
+
+        ]
+
+
+    // MARK: - Table view data source
+
+
+
+    override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        // #warning Incomplete implementation, return the number of rows
+        return vocabWords.count"
```

Comment:
you don't need the " at the end. maybe this just a typo.

Reply:
This also gives you an error "Untermined string literal".

Probably because above you have your vocabWords array in your viewDidLoad()